alchemist/src/orchid/resources/wiki/usage/maps.md

```diff
@@ -0,0 +1,87 @@
+---
+
+title: Maps and GPS traces
+
+---
+
+Alchemist is equipped with the ability to load and simulate on real-world maps.
+Navigation on maps can be done by using gps traces,
+by moving along roads (Alchemist relies on [GraphHopper](https://www.graphhopper.com/) to provide directions),
+by interpolating gps traces with on-the-road-movements,
+or by ignoring the map information on just move as you would in a continous space.
+
+Configuring a simulation to run on a map, possibly using GPS tracks, requires three parameters to be specified:
+0. Environment
+0. Displacement
+0. Program to follow the GPS traces
+
+## Setting up a map environment
+
+In order to run simulations on real world maps, an appropriate environment must be selected.
+
+Currently, the only environment supporting maps is {{ anchor('OSMEnvironment') }}. 
+
+The only mandatory parameter is the path of the file with the map to be loaded.
+For the optional parameters, refer to the documentation of the environment.
+
+The following example shows how to configure a simulation that loads data from an Openstreetmap file 
+(OSM, XML and PBF formats are supported) located in the classpath folder `maps`:
+
+```yaml
+incarnation: protelis
+environment:
+  type: OSMEnvironment
+  parameters: [maps/foo.pbf]
+```
+
+## Using GPS traces to displace nodes
+
+GPS traces can be used to displace nodes on a map.
+In order to set the initial position of the nodes with the first position of the GPS traces,
+you need to use the displacement {{ anchor('FromGPSTrace') }}.
+The displacement `FromGPSTrace` require the following parameters:
+
+0. number of nodes to displace
+0. file with the GPS traces to use
+0. boolean that indicates if the list of GPS trace is cyclic.
+   If `true` and the GPS traces are less than the number of nodes to be displaced,
+   then the traces are cyclically re-used to displace nodes.
+0. a {{ anchor('GPSTimeAlignment') }} to define how to align the time of all the GPS points of all GPS traces. 
+    There are present several strategy in the package {{ anchor('it.unibo.alchemist.boundary.gpsload.api') }}
```

Comment:
Add a subsection with a description of the available `GPSTimeAlignment`s

Reply:
added